load_balancer/status.js

```diff
@@ -0,0 +1,102 @@
+/* eslint import/no-dynamic-require: 0 */
```

Comment:
looks like this module can make use of lodash module to simplify the logic. Please see if you can use it to reduce the code.

Reply:
May be its just me, but the code does not look readable to me. Is there a way to simplify the code?

Reply:
I reviewed the lodash vs es6 debate. Its definitely worth using lodash in the project. I would rather introduce lodash more strategically by identifying the features of lodash that help us, note them down for use during refactoring. Nothing much to do now.